adafruit_framebuf.py

```diff
@@ -352,9 +361,10 @@ def __exit__(self, exception_type, exception_value, traceback):
         """cleanup on exit"""
         self.deinit()
 
-    def draw_char(self, char, x, y, framebuffer, color):
-        # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments
+    def draw_char(self, char, x, y, framebuffer, color, size=1):
```

Comment:
add the disable to the end of the def line and it will be scoped to this function only. Otherwise put an enable after the function.